pkg/elemental/elemental.go

```diff
@@ -294,16 +294,16 @@ func UnmountFileSystemImage(c v1.Config, img *v1.Image) error {
 // CreateFileSystemImage creates the image file for the given image. An root tree path
 // can be used to determine the image size and the preload flag can be used to create an image
 // including the root tree data.
-func CreateFileSystemImage(c v1.Config, img *v1.Image, rootDir string, preload bool) error {
-	c.Logger.Infof("Creating image %s", img.File)
+func CreateFileSystemImage(c v1.Config, img *v1.Image, rootDir string, preload bool, excludes ...string) error {
```

Comment:
I am wondering if do we have a need to actually bring this excluding paths up that far. Do we have any call of `CreateImageFromTree` where we don't set it to the defaults or were this could have an undesired effect? Probably the same applies for `CreateFileSystemImage`.

I mean probably it is just enough adding the default excludes inside  `CreateFileSystemImage` and on `CreateImageFromTree` and there is no need to map this change that far up. I think having it ported all way up males sense if we make use of it at action level, otherwise probably we can keep it minimal.

Reply:
Good point. I didn't want to sneak it in because I was afraid it could have unexpected effects. 
This is also used by the build-disk, however I agree it *should* not have any undesired effect. 

I think we can hardcode this in CreateImageFromTree, since we need to pass the exclusion list to CreateFileSystemImage and later utils.SyncData.